src/commonMain/kotlin/fr/acinq/lightning/channel/Closing.kt

```diff
@@ -368,6 +369,7 @@ data class Closing(
                     when (it) {
                         is Transactions.TransactionWithInputInfo.ClaimHtlcTx.ClaimHtlcSuccessTx -> add(it.tx to "remote-htlc-success")
                         is Transactions.TransactionWithInputInfo.ClaimHtlcTx.ClaimHtlcTimeoutTx -> add(it.tx to "remote-htlc-timeout")
+                        else -> Unit
```

Comment:
same as above

Reply:
Done in https://github.com/ACINQ/lightning-kmp/commit/d75e77b588fef36607f8b5639cb34e7d1fcaf660.